android/capacitor/src/main/java/com/getcapacitor/JSObject.java

```diff
@@ -40,32 +41,37 @@ public static JSObject fromJSONObject(JSONObject obj) throws JSONException {
     }
 
     @Override
+    @Nullable
     public String getString(String key) {
         return getString(key, null);
     }
 
-    public String getString(String key, String defaultValue) {
+    @Nullable
+    public String getString(String key, @Nullable String defaultValue) {
         try {
             String value = super.getString(key);
-            if (!super.isNull(key) && value != null) {
+            if (!super.isNull(key)) {
```

Comment:
I remove the null check here because the `super` call actually throws an exception if the value is null so it was a redundant call. Android studio was showing a warning for it.